docs/content/components/file_tree.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: FileTree
+description: A file tree component for displaying directory structures.
+---
+
+---
+
+File Trees are a great way to display directory structures in your site. It renders a collapsible tree view of files and folders, and automatically shows icons for different file types.
+
+## Usage
+
+Add the `FileTree()` component to your `components` list in your `main.dart` file:
+
+```dart
+import 'package:jaspr_content/components/file_tree.dart';
+
+// ...
+
+ContentApp(
+  components: [
+    FileTree(),
+  ],
+)
+```
+
+Then use the `<FileTree>` component in your content files to display the file trees. Specify the structure of your files and directories with an unordered Markdown list. Create a sub-directory using a nested list or add a `/` to the end of a list item to render it as a directory without specific content.
+
+```markdown
+<FileTree>
+
+- lib/
+  - components/
+    - clicker.dart
+  - **main.dart** Highlighted file with comment
+- tool/
+- .gitignore
+- analysis_options.yaml
+- CHANGELOG.md
+- LICENSE 
+- package.yaml 
+- README.md This is a comment
+
+</FileTree>
+```
+
+- You can highlight specific files or folders by making the text bold using `**` or `__`.
+
+- You can add comments to files or folders by adding text after the name of the file or folder.
+
+- You can also specify folders to be closed by default by prefixing the folder name with a `^` character.
```

Comment:
Wooo, thanks for adding this! I think this syntax works well and makes decent sense just reading it.